packages/brokeneck-react/src/components/ErrorBoundary.test.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import { screen, render } from '@testing-library/react'
+
+import ErrorBoundary from './ErrorBoundary'
+
+const ThrowError = ({ shouldThrow }) => {
+  if (shouldThrow) {
+    throw new Error('Custom error message')
+  } else {
+    return null
+  }
+}
+
+beforeEach(() => {
+  // eslint-disable-next-line no-console
+  console.error = jest.fn()
+})
+
+describe('ErrorBoundary', () => {
+  it('should render ErrorBoundary component when child throws error', () => {
+    const { rerender } = render(<ThrowError />, { wrapper: ErrorBoundary })
+
+    rerender(<ThrowError shouldThrow={true} />)
```

Comment:
why render then rerender? can't you render once with shouldThrow=true to test this scenario?